### PR TITLE
rspec: add Indonesian translation

### PR DESCRIPTION
**Pull request by apraditya**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/common/rspec.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`rspec --init`
+
+- Menjalankan semua file tes:
+
+`rspec`
+
+- Menjalankan file tes dalam direktori khusus:
+
+`rspec {{lokasi/ke/directory}}`
```
Comment (kbdharun) [inline pages.id/common/rspec.md:16]:
```suggestion
`rspec {{jalan/menuju/direktori}}`
```

File: pages.id/common/rspec.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Menjalankan file tes khusus:
+
+`rspec {{lokasi/ke/file}}`
```
Comment (kbdharun) [inline pages.id/common/rspec.md:20]:
```suggestion
`rspec {{jalan/menuju/file}}`
```

File: pages.id/common/rspec.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Menjalankan beberapa file tes:
+
+`rspec {{lokasi/ke/file1}} {{lokasi/ke/file2}}`
```
Comment (kbdharun) [inline pages.id/common/rspec.md:24]:
```suggestion
`rspec {{jalan/menuju/file1}} {{jalan/menuju/file2}}`
```

File: pages.id/common/rspec.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Menjalankan kasus khusus dalam file tes (misalnya tes yang ada di baris 83):
+
+`rspec {{lokasi/ke/file}}:{{83}}`
```
Comment (kbdharun) [inline pages.id/common/rspec.md:28]:
```suggestion
`rspec {{jalan/menuju/file}}:{{83}}`
```

File: pages.id/common/rspec.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Menjalankan tes dengan _seed_ khusus:
+
+`rspec --seed {{seed_number}}`
```
Comment (kbdharun) [inline pages.id/common/rspec.md:32]: Translate `seed_number` command argument too for better readability to native readers.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by navarroaxel — APPROVED** [review]

File: pages.id/common/rspec.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`rspec --init`
+
+- Menjalankan semua file tes:
+
+`rspec`
+
+- Menjalankan file tes dalam direktori khusus:
+
+`rspec {{jalan/ke/directory}}`
```
Comment (kbdharun) [inline pages.id/common/rspec.md:16]:
```suggestion
`rspec {{jalan/menuju/direktori}}`
```

File: pages.id/common/rspec.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+- Menjalankan file tes khusus:
+
+`rspec {{jalan/ke/file}}`
```
Comment (kbdharun) [inline pages.id/common/rspec.md:20]:
```suggestion
`rspec {{jalan/menuju/file}}`
```

File: pages.id/common/rspec.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+- Menjalankan beberapa file tes:
+
+`rspec {{jalan/ke/file1}} {{jalan/ke/file2}}`
```
Comment (kbdharun) [inline pages.id/common/rspec.md:24]:
```suggestion
`rspec {{jalan/menuju/file1}} {{jalan/menuju/file2}}`
```

File: pages.id/common/rspec.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+- Menjalankan kasus khusus dalam file tes (misalnya tes yang ada di baris 83):
+
+`rspec {{jalan/ke/file}}:{{83}}`
```
Comment (kbdharun) [inline pages.id/common/rspec.md:28]:
```suggestion
`rspec {{jalan/menuju/file}}:{{83}}`
```

File: pages.id/common/rspec.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+- Menjalankan tes dengan seed khusus:
+
+`rspec --seed {{angka_seed}}`
```
Comment (kbdharun) [inline pages.id/common/rspec.md:32]: I have a doubt `angka_seed` means number_seed right? Shouldn't it be reversed?

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages.id/common/rspec.md (hunk 10 above)
Comment (apraditya) [reply pages.id/common/rspec.md:32]: Thanks for pointing this out. Appreciate the attention to the detail like this.

The argument expects a number. To communicate it clearly to the user,

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.id/common/rspec.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
+# rspec
+
+> Kerangka pengujian dalam Behavior-driven development yang ditulis dalam bahasa Ruby untuk menguji kode Ruby.
+> Informasi lebih lanjut: <https://rspec.info>.
+
+- Menginisiasi file konfigurasi .rspec dan spec helper:
```
Comment (kbdharun) [inline pages.id/common/rspec.md:6]:
```suggestion
- Menginisiasi file konfigurasi `.rspec` dan spec helper:
```

**Pull request merged**